### PR TITLE
Layout + charts

### LINKED ISSUE #21: Make it pretty
- [x] Edit layout to more closely match wireframe and inspirations
    - [x]  Make drop down's horizontal
    - [x] Add user Icon
    - [x] Attempt to use Chart.js package to display data

### PR DESCRIPTION
**Pull request by Andrew-Musselman**
closes issue #21 

# Description

In this pull request I have updated our hydration card to only display the charts that are now being rendered with the users data. I also made some layout changes in the css file to properly display the charts on the page. I cleaned up the html a bit and got rid of the unused sections that were displaying the data before the charts were functioning.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
    - Changed: Rendered charts based on current user. User now generates randomly on page load. 
    - Bugs created: none
- [ ] Bug fix (non-breaking change which fixes an issue)
    -Bug fixed:
- [ ] Refactor
    -Code refactoredL=:
- [ ] Documentation update

## How Has This Been Tested?

All changes were tested by viewing the page that was running through local server. 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have fully tested my code
- [x] My code follows the style guidelines of this project.
- [x] My code has no unused/commented-out code.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

## Future Steps:

The next step would be to create functionality for the date pickers to allow the user to change the data being shown in the charts

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -2,42 +2,42 @@
...
   <head>
     <meta charset="UTF-8">
-    <title>Fitlit</title>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+    <title>HealthDash</title>
   </head>
   <body>
     <nav>
-      <h1>Activity Tracker</h1>
+      <h1>HealthDash</h1>
```
Comment (BrendaSotel0) [inline dist/index.html:10]: Great name!

Comment (zliibbe) [reply dist/index.html:10]: I like the new name

File: dist/index.html [hunk 2]
```diff
@@ -2,42 +2,42 @@
...
-            <p id="daily-sleep">daily</p>
             <section id="daily-sleep-quality-data">
               <p></p>
             </section>
             <section id="daily-sleep-quantity-data">
               <p></p>
             </section>
             <p id="weekly-sleep">weekly</p>
-            <section id="weekly-sleep-quality-data">
+            <section class="weekl-view">
```
Comment (BrendaSotel0) [inline dist/index.html:40]: Looks like this might be misspelled.

**Review by BrendaSotel0 — COMMENTED** [review]
Good job.

**Review by zliibbe — APPROVED** [review]
Looks good overall

**Pull request merged**